alis.sh

```diff
@@ -747,7 +756,7 @@ function mkinitcpio_configuration() {
     if [ "$LVM" == "true" ]; then
         HOOKS=${HOOKS//!lvm2/lvm2}
     fi
-    if [ "$BOOTLOADER" == "systemd" ]; then
+    if [ "$BOOTLOADER" == "systemd" ] || [ "$GPT_AUTOMOUNT" == "true" ]; then
```

Comment:
Is necessary add this or condition?

Reply:
Thinking a second time about it: Only if the root partition is encrypted.

According to https://www.freedesktop.org/software/systemd/man/systemd-gpt-auto-generator.html: "When systemd is running in the initrd the / partition may be encrypted in LUKS format as well." 
Therefore, if the root partition is LUKS encrypted. it is necessary that, instead of busybox init, systemd init is employed.

I forgot to add the check whether the root partition should be encrypted.

Reply:
I have updated my branch correspondingly.